docs/sources/operator/deploy-agent-operator-resources.md

```diff
@@ -58,7 +58,7 @@ To deploy the `GrafanaAgent` resource:
       labels:
         app: grafana-agent
     spec:
-      image: grafana/agent:v0.35.4
+      image: grafana/agent:v0.36.0
```

Comment:
Same for all the other references, too ([sample PR](https://github.com/grafana/agent/pull/3065/files))